<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopBtn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QWidget" name="menu" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#menu{
	background-color: #aaaa7f;
}</string>
    </property>
    <widget class="QWidget" name="dashboard" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget:hover{
	background-color: #87CBB9;
}</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>-4</x>
        <y>3</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>dashboard</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="data" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget:hover{
	background-color: #87CBB9;
}</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>-4</x>
        <y>3</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>data</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="table1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>531</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
